Automattic/jetpack
# AI Client: Filter suggestions starting with llama artifacts

ref p1720130920711329/1718723268.245799-slack-C050S8KG677

## Proposed changes:

* Removes the starting `<|start_header_id|>assistant<|end_header_id|>\n` text from llama suggestions

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Sandbox the public API and simulate a llama3 call on Breve (ref p1720134386106669/1720134295.868489-slack-C054LN8RNVA)
* Without this PR applied, check that changes turn into the llama artifact string
* With this PR applied, check that the requests work as expected
